src/commands/e2e/E2E.ts

```diff
@@ -30,8 +30,8 @@ export class E2E implements ICommand {
     private static readonly PARAMETER_SCREENSHOT: string = 'screenshot';
 
     // Default
-    private static readonly DEFAULT_BASE_URL: string = 'http://localhost:8083';
-    private static readonly DEFAULT_CONTEXT: string = '/intern/tricia/';
+    private static readonly DEFAULT_BASE_URL: string = 'http://localhost:8083/';
```

Comment:
Please don't change these - the slashes match the requirements posed by the `configuration.txt` properties. We should make sure that the settings here really match this (as the checks below really forced that).